src/formatDate.js

```diff
@@ -50,7 +50,42 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const formatYear = function(year) {
+    if (fromFormat.includes('YY') && toFormat.includes('YYYY')) {
+      if (year < 30) {
+        return '20' + year;
+      } else {
+        return '19' + year;
+      }
+    } else if (fromFormat.includes('YYYY') && toFormat.includes('YY')) {
+      return year.slice(-2);
+    }
+
+    return year;
+  };
+
+  const separatorBefore = fromFormat[3];
+  const separatorAfter = toFormat[3];
+  const dateInArray = date.split(separatorBefore);
+  const correctDate = [];
```

Comment:
maybe better newDate?

Reply:
Yes, it is better describes what I wanted to put there, thanks